weave/panels/panel_table.py

```diff
@@ -325,9 +325,10 @@ def rows_single_refine(self: Table) -> weave.types.Type:
     return weave.types.optional(_get_row_type(self))
 
 
+# TODO: preserve arrow
```

Comment:
This PR results in faster loading of a notebook test. As a result, the `table_data_passing` test started failing (it errors on master, but too slowly to be caught by the test). This is a "fix" that ensures the type is correctly marked as a list. However, like all these functions, there is still a follow-up to make these arrowified.